controller/controller.go

```diff
@@ -445,6 +445,34 @@ func StartInformers(stopCh <-chan struct{}, informers ...Informer) error {
 	return nil
 }
 
+// RunInformers kicks off all of the passed informers and then waits for all of
+// them to synchronize. Returned channel will get closed after all informers
+// finish running.
+func RunInformers(stopCh <-chan struct{}, informers ...Informer) (<-chan struct{}, error) {
```

Comment:
`RunWithWait/RunAndWait`?

Reply:
I pushed a new implementation using `errgroup.Group` before having a chance to see those comments. Sorry about that.

I don't have strong opinion about the name as long as this does unflake the data race.